geonode/documents/views.py

```diff
@@ -489,9 +490,30 @@ def document_metadata_detail(request, docid, template='documents/document_metada
 @login_required
 def document_list(request):
     documents = Document.objects.all()
+    filename = str(datetime.datetime.now()) + "_documents.csv"
     response = HttpResponse(content_type='text/csv')
-    response['Content-Disposition'] = 'attachment; filename="geonode_documents.csv"'
+    response['Content-Disposition'] = 'attachment; filename=' + filename
     writer = csv.writer(response)
+    # Writes header row
+    writer.writerow(['Layer ID',
+                    'Title',
```

Comment:
Wrong indentation here : you should indent `'Title'` and next lines to `'Layer ID'` so you need to add one space.